release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1145,6 +1145,10 @@ With this release, replacement control plane nodes are assigned to the correct i
 [id="ocp-4-13-olm-bug-fixes"]
 ==== Operator Lifecycle Manager (OLM)
 
+* The status of an Operator’s custom resource (CR) includes a list of components owned by the Operator. This list is ordered by group/version/kind (GVK), but the order of objects with the same GVK might change. If an Operator owns many components with the same GVK, it can cause Operator Lifecycle Manager (OLM) to continuously update the status of the Operator CR, because the order of its components has changed. This bug fix updates OLM so that the order of an Operator's component references is deterministic. As a result, OLM no longer attempts to update the CR repeatedly when the list of components remains constant. (link:https://issues.redhat.com/browse/OCPBUGS-2556[*OCPBUGS-2556*])
```

Comment:
I am not sure about "group/version/kind (GVK)" I was looking at the ISG for reference https://www.ibm.com/docs/en/ibm-style?topic=punctuation-slashes and I am not sure if it falls under the "where the slash construction is an industry standard" section. It may be correct the way it is, but I am not positive.

Reply:
I experienced the use of this term during my RH OpenJDK stint and `grup/version/kind` was correct. I did see `GroupVersionKind` mentioned in the following K8s guide, if this would work?

https://book.kubebuilder.io/cronjob-tutorial/gvks.html

Reply:
Then again, I found the following in published OCP docs:

![Screenshot from 2023-04-19 16-46-59](https://user-images.githubusercontent.com/57954076/233130016-d4adb839-72a3-4dad-9073-5334bc54b98b.png)

Link: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/operators/understanding-operators

Reply:
I placed backticks around `group/version/kind` so that this highlighting would indicate it's a definite term and not a choice of three options or three ways to name one thing.

Reply:
Yeah backticks makes sense to me. Thanks!